tests/python/unittest/test_gluon_probability.py

```diff
@@ -0,0 +1,2326 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import tempfile
+
+import mxnet as mx
+from mxnet import np, npx, autograd
+from mxnet import gluon
+import mxnet.gluon.probability as mgp
+from mxnet.gluon.probability import StochasticBlock, StochasticSequential
+from mxnet.gluon import HybridBlock
+from mxnet.test_utils import use_np, assert_almost_equal, set_default_context
+import numpy as _np
+from common import (setup_module, with_seed, assertRaises,
+                    assert_raises_cudnn_not_satisfied)
+from numpy.testing import assert_array_equal
+import pytest
+import scipy.stats as ss
+import scipy.special as scipy_special
+import warnings
+import json
+import unittest
+import random
+import itertools
+from numbers import Number
+
+# set_default_context(mx.gpu(0))
```

Comment:
You should delete this line here and create a separate file in the `gpu` test folder. In that file, import all functions from the current `test_gluon_probability.py` file

Reply:
Oh, I forgot that, I will change it soon.